src/wx/openal.cpp

```diff
@@ -356,10 +356,10 @@ bool OpenAL::GetDevices(wxArrayString& names, wxArrayString& ids)
         devs += strlen(devs) + 1;
     }
 
-#else
+#endif
+
     // should work anyway, but must always use default driver
     return true;
```

Comment:
This pattern ensures the compiled code will always have `return true`. Technically both are fine, but the new way won't cause the linter to choke since the logic isnt totally hidden inside the `#` conditionals.